ooniprobe/Utility/OONIApi.h

```diff
@@ -0,0 +1,9 @@
+#import <Foundation/Foundation.h>
+
+@interface OONIApi : NSObject
+
++ (void)downloadUrls:(void (^)(NSArray*))successcb onError:(void (^)(NSError*))errorcb;
++ (void)downloadJson:(NSString*)urlStr onSuccess:(void (^)(NSDictionary*))successcb
+onError:(void (^)(NSError*))errorcb;
```

Comment:
Please indent the on onError so that there are at least 4 whitespaces after the beginning of the line

Reply:
Done. Also I forgot a commit that wasn't pushed.